### DISCUSSION TITLE
win10和taskbarX一起使用会导致taskbar闪烁

### DISCUSSION THREAD

**Discussion by yunyuyuan** [zhongyang219/TrafficMonitor#1175] [Q&A]
windows10，不知道有没有人在用taskbarX让任务栏居中，TrafficMonitor的任务栏窗口会导致taskbarX偶尔横跳，关掉任务栏窗口就好了。有没有让它们共存的方案呢？

**Comment by coder-user** [comment]
我的也是这样的，请问你最后是怎么处理的？

**Comment by yunyuyuan** [comment]
@coder-user 我更新win11了。。。
